### ISSUE TITLE
Metadata cleanup

### ISSUE THREAD

**Issue by jarad** [jarad/ISUmonarch#27]
I think we need to clean up the site metadata. My proposal is to create 1) a single csv file for transect information that does not change over time, 2) a single csv file for site information that does not change over time, and 3) individual files for round in a year. Here are some questions I have that I can't answer based on the data that I can see in the Site Metadata file:

- how many rounds were performed in 2017? 
- when were the rounds performed in 2017? *
- what is r3_date_resample?

Basically the idea is to create files that once set don't have to be touched, e.g. the 2016 data and now the 2017 data. 

* This question is pertinent to create appropriate NCIG tables. I believe there were 3 rounds for every transect, but that isn't immediately clear.

**Comment by sethapp** [comment]
I uploaded 3 new documents. Let me know if you think this resolves the issue. The 'resample' means that for whatever reason we had to redo some of the data at a site on a different day (for example: we forgot to collect a data piece or it was raining and we couldn't do a monarch survey). but we only resample the data that needs to be resampled, not all of it.

**Comment by sethapp** [comment]
I created a new 'metadata' folder with years inside it. I renamed the original files and converted to CSV. Do I need to resolve other issues for them or are they ok? I currently left the originals in the 'data-raw' folder until I know I've done this process correctly. Once I know I haven't messed something up I'll delete them.

Also, still in the data raw folder: there are two 'site metadata' files. One is outdated, can I delete it? Are there any scripts tied to it? The 'Site Metadata Git- Site info' is the current file (it needs some updating still). 

Lastly, do I need to convert the site metadata file to a CSV or is it fine as is?

**Comment by jarad** [comment]
Definitely you can delete any files that are outdated. As a reminder when you delete files they still exist in the history of the repository, i.e. they can be recovered at any time. So there really is no issue with deleting the original meta-data files as you can always recover them.

We should probably write a script to read in the metadata, but at the moment, it isn't clear to me what we want out of the meta-data.

**Comment by sethapp** [comment]
I believe the new 'survey' folder and other recent updates have fixed this issue
